recipes/_configure_server.rb

```diff
@@ -133,33 +133,49 @@
 end
 
 # License Configuration
+node.run_state['cerner_splunk'] ||= {}
+
 license_uri =
   case node['splunk']['node_type']
-  when :forwarder, :license_server
+  when :license_server
     'self'
   when :cluster_master, :cluster_slave, :server, :search_head, :shc_search_head, :shc_captain, :shc_deployer
     if node['splunk']['free_license']
       'self'
     else
       CernerSplunk.my_cluster_data(node)['license_uri'] || 'self'
     end
+  when :forwarder
+    if node.run_state['cerner_splunk']['heavy_forwarder']
+      CernerSplunk.my_cluster_data(node)['license_uri'] || 'self'
```

Comment:
I keep thinking about a node attribute here... If set to true -> use cluster data license_uri... otherwise self... 

Having an attribute that has to be set to true for a HWF to use the license_uri, keeps passivity with existing versions, and gives the option for those that need it to set it. 

I also think about the same attribute possibly being able to set an override license_uri... which would enable us to lock down the license master more, and shove all the forwarders off to a different LM (possibly).